wpilib/tests/test_solenoid.py

```diff
@@ -92,10 +102,12 @@ def test_solenoidbase_getAll(wpilib, hal_data):
         s['value'] = True
         
     assert solenoid.getAll() == 0xFF
+    assert wpilib.SolenoidBase.getAll(0) == 0xFF
```

Comment:
the getAll weirdness: upstream they added a static getAll with a parameter, but left the old instance method with no parameter, so to get the same behavior in python requires some hacking